infrastructure/terraform/modules/browser/dns.tf

```
@@ -0,0 +1,47 @@
+
+resource "cloudflare_zone" "dns" {
+    zone = "${var.domain}"
```

No need for "string interpolation" -- you only need it if you are doing concatenation variable with a string.

```suggestion
    zone = var.domain
```